src/DataTable/DataTable.svelte

```diff
@@ -1,39 +1,41 @@
 <script>
   /**
-   * @typedef {string} DataTableKey
+   * @generics {Row extends DataTableRow = DataTableRow} Row
+   * @template {DataTableRow} Row
+   * @typedef {Exclude<keyof Row, "id">} DataTableKey<Row=DataTableRow>
```

Comment:
The key type must disallow `"id"` being used as value, as that is a reserved prop for `rows`.

Reply:
I don't see a reason why the `id` would not also be displayable as a column?

Reply:
Good point